packages/webdriverjs/README.md

```diff
@@ -36,140 +36,115 @@ driver.get('https://dequeuniversity.com/demo/mars/').then(function () {
+Limits analysis to only those with the specified rule IDs. Accepts a String of a single tag or an Array of multiple tags. Subsequent calls to `AxeBuilder#options`, `AxeBuilder#withRules` or `AxeBuilder#withRules` will override specified options.
 
-```javascript
-new AxeBuilder(driver).withTags('wcag2a');
+```js
+new AxeBuilder({ driver }).withTags('wcag2a');
 ```
 
-```javascript
-new AxeBuilder(driver).withTags(['wcag2a', 'wcag2aa']);
+```js
+new AxeBuilder({ driver }).withTags(['wcag2a', 'wcag2aa']);
 ```
 
-### AxeBuilder#disableRules(rules:Mixed)
+### AxeBuilder#disableRules(rules: String|Array)
 
-Skips verification of the rules provided. Accepts a String of a single rule ID or an Array of multiple rule IDs. **Subsequent calls to `AxeBuilder#options`, `AxeBuilder#disableRules` will override specified options.**
+Skips verification of the rules provided. Accepts a String of a single rule ID or an Array of multiple rule IDs. Subsequent calls to `AxeBuilder#options`, `AxeBuilder#disableRules` will override specified options.
 
-```javascript
-new AxeBuilder(driver).disableRules('color-contrast');
+```js
+new AxeBuilder({ driver }).disableRules('color-contrast');
 ```
 
-or use it combined with some specified tags:
+### AxeBuilder#configure(config: [axe.Spec](https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#api-name-axeconfigure))
 
-```javascript
-new AxeBuilder(driver)
-  .withTags(['wcag2a', 'wcag2aa'])
-  .disableRules('color-contrast');
-```
-
-### AxeBuilder#configure(config:Object)
+Inject an axe configuration object to modify the ruleset before running Analyze. Subsequent calls to this method will invalidate previous ones by calling `axe.configure` and replacing the config object. See [axe-core API documentation](https://github.com/dequelabs/axe-core/blob/master/doc/API.md#api-name-axeconfigure) for documentation on the object structure.
 
-Inject an aXe configuration object to modify the ruleset before running Analyze. Subsequent calls to this
-method will invalidate previous ones by calling `axe.configure` and replacing the config object. See
-[axe-core API documentation](https://github.com/dequelabs/axe-core/blob/master/doc/API.md#api-name-axeconfigure)
-for documentation on the object structure.
+```js
+const config = {
+  checks: axe.Check[],
+  rules: axe.Rule[]
+}
 
-```javascript
-var config = {
-  checks: [Object],
-  rules: [Object]
-};
-
-new AxeBuilder(driver).configure(config).analyze(function (err, results) {
+new AxeBuilder({ driver }).configure(config).analyze((err, results) => {
   if (err) {
     // Handle error somehow
   }
-  console.log(results);
-});
+  console.log(results)
+})
 ```
 
-### AxeBuilder#analyze(callback:Function)
+### AxeBuilder#analyze(): Promise<axe.Results | Error>
```

Comment:
This reads as:

The class `AxeBuilder`'s method `analyze` returns a `Promise` which resolves to `axe.Results` or an `Error`, which is not correct. The `Promise` will never _resolve_ with an error, but it may `reject` instead.

IMO this should be:

```suggestion
### AxeBuilder#analyze(): Promise<axe.Results>
```